.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,37 @@
+// For format details, see https://aka.ms/devcontainer.json.
+{
+    "name": "compare_chain",
+    // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+    "image": "elixir:1.16-otp-26-slim",
+
+    // Features to add to the dev container. More info: https://containers.dev/features.
+    "features": {
+      "ghcr.io/devcontainers/features/common-utils:2": {
+        "installOhMyZsh": false,
+        "installOhMyZshConfig": false
+      },
+      "ghcr.io/devcontainers/features/git:1": {
+        "version": "latest"
+      },
+      "ghcr.io/devcontainers/features/github-cli:1": {}
+    },
+
+    // Use 'forwardPorts' to make a list of ports inside the container available locally.
+    // "forwardPorts": [],
+
+    // Use 'postCreateCommand' to run commands after the container is created.
+    "postCreateCommand": ".devcontainer/setup.sh",
+
+    // Configure tool-specific properties.
+    "customizations": {
+        "vscode": {
+            "extensions": [
+              "EditorConfig.EditorConfig",
+              "JakeBecker.elixir-ls"
```

Comment:
Are there other popular Elixir extensions for VS Code that would be candidates for inclusion here?

Reply:
No I think these are sufficient.